### PR TITLE
Read include_languages and exclude_languages from config

### PR DESCRIPTION
**Pull request by vivianrdu**

### CODE REVIEW

**Comment by vivianrdu** [comment]
I'm not quite sure how WikipediaSpider should access the include_languages/exclude_languages functions from load_configs.py -- or if this is the correct approach?

**Comment by smrohrer** [comment]
We could call `read_include_languages` and `read_exclude_languages` in language_indexing.py. Then we pass the return values into the `WikipediaSpiderInput` constructor [here](https://github.com/wikitongues/Language-Indexing/blob/f34ca390d8e044d9f9c01cd9031f5c1735414bf4/wikitongues/wikitongues/language_indexing.py#L49). This way, we are using [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection) to provide the WikipediaSpider with the information it needs

File: wikitongues/wikitongues/spiders/wikipedia_spider.py [hunk 1]
```diff
@@ -26,8 +28,11 @@ def __init__(self, spider_input, language_data_store, *args, **kwargs):
 
     # Load Language objects to target in this crawl
     def load_languages(self):
-        result = self._language_data_store.get_languages(
-            self._spider_input.iso_codes)
+        if self._spider_input.iso_codes is not None:
+            result = self._language_data_store.get_languages(
+                self._spider_input.iso_codes)
+        else:
+            result = self._language_data_store.list_languages()
```
Comment (smrohrer) [inline wikitongues/wikitongues/spiders/wikipedia_spider.py:35]: Use `self._spider_input.exclude_iso_codes` to filter `result` here

**Review by smrohrer — CHANGES_REQUESTED** [review]

**Comment by vivianrdu** [comment]
For `read_include_languages(config)`, if include_languages isn't provided in the user_config, should the function return all possible iso_codes?

**Comment by smrohrer** [comment]
No, it should just return `None`. In that case the WikipediaSpider will use the data store to list data objects for all languages (subject to the configured page size and offset), instead of getting data objects for specific languages

File: wikitongues/wikitongues/config/load_configs.py [hunk 2]
```diff
@@ -92,3 +92,21 @@ def load_languages_airtable_datastores(config):
...
+
+
+def read_include_languages(config):
+
+    if len(config.items("include_languages")) > 0:
+        return config['include_languages']
+
+    iso_codes = [
+        iso for iso in list(config._sections['language_codes'].values())
+    ]
```
Comment (smrohrer) [inline wikitongues/wikitongues/config/load_configs.py:104]: This part shouldn't be necessary, if we are assuming `include_languages` is a comma-separated list of iso codes. We'll just need to use [`split()`](https://www.w3schools.com/python/ref_string_split.asp) somewhere to convert the string value from the config into a list.

**Review by smrohrer — CHANGES_REQUESTED** [review]

File: wikitongues/wikitongues/config/load_configs.py (hunk 2 above)
Comment (vivianrdu) [reply wikitongues/wikitongues/config/load_configs.py:104]: Sounds good!

**Review by smrohrer — APPROVED** [review]

**Pull request merged**